roles.json

```
@@ -395,7 +395,24 @@
         }
     },
     {
-        "role": "Sub-package maintainer",
+        "role": "Core astropy package maintainer (general)",
+        "url": "Core_package_general_maintainer",
+        "people": [
+            "unfilled"
+        ],
+        "role-head": "Core astropy package maintainer (general)",
+        "responsibilities": {
+            "description": "Maintain the astropy core package in aspects that are not specific to a single sub-package",
+            "details": [
+                "Evaluating new pull requests for quality, API consistency, Astropy coding standards, and appropriateness within the overall astropy ecosystem, in particular for pull requests spanning multiple sub-packages",
+                "Merging Pull Requests",
+                "Maintain, review, and advocate for useful interaction between multiple sub-packages", 
+                "Keeping track of frequent contributors and their relevant areas of expertise"
```

In the telecon there was some discussion of the need to distinguish this from the devops role. Looking at the exact wording I do see some overlap here, particularly with the "Merge non-controversial pull requests" aspects of DevOps.

Concretely I think maybe a change like this is warrented:
```suggestion
                "Evaluating new pull requests for quality, API consistency, Astropy coding standards, and appropriateness within the overall astropy ecosystem, in particular for pull requests spanning multiple sub-packages",
                "Merging Pull Requests that are non-controversial or in consultation with relevant subpackage maintainers",
                "Maintain, review, and advocate for useful interaction between multiple sub-packages", 
                "Keeping track of frequent contributors and their relevant areas of expertise"
```

I *also* think there's some question of how much of this work is something the DevOps role does as part of triaging/labeling/etc. That is, in practicle I think the "keeping track" may be more logically connected to the triaging/labeling process. But maybe that really means that most devops folks should *also* be generalists?  Or should we add this last line to the devops role?

---

Alternative wording:
```suggestion
                "Evaluating new pull requests for quality, API consistency, Astropy coding standards, and appropriateness within the overall astropy ecosystem, in particular for pull requests spanning multiple sub-packages",
                "Merging Pull Requests that are non-controversial or after reaching out to relevant subpackage maintainers",
                "Maintain, review, and advocate for useful interaction between multiple sub-packages", 
                "Keeping track of frequent contributors and their relevant areas of expertise"
```

---

After zoom discussion with @eteq we agreed on this wording. For reference: Our thought is that we want the generalists to merge stuff that's uncontroversial, just like we (Eric and I) want other sub-package maintainers to merge PRs outside their sub-pacakges that are clearly uncontroversial (typos in the docs would be an extreme example,  other PRs are that clearly uncontroversial in the sense that they are discussed on the PR in detail and everyone agrees also fall into this category), but also PRs that go beyond that. The language "after reaching out" is supposed to mean that sub-package maintainers should have a chance to weigh in, but if they don't reply after a reasonable time, or if a sub-package does not have a maintainer, the generalist can merge the PR. Please comment on the wording if you havea better idea on how to say it.